### PR TITLE
csv : Complete check if row is empty (carriage return)

### PR DESCRIPTION
**Pull request by benoit-bst**

### CODE REVIEW

File: csv.cpp [hunk 1]
```diff
@@ -258,3 +258,16 @@ void remove_bom(std::fstream& stream){
...
+ * Avoid to process empty row and row with only carriage return characters.
+ *
+ * @return true if row is empty
+ */
+bool CsvReader::row_is_empty(const std::vector<std::string>& row) {
+    if(row.empty() || (row.size() == 1 && row[0].empty()))
+        return true;
+    else
+        return false;
+}
```
Comment (TeXitoi) [inline csv.cpp]: Lack empty line at the end of the file.

Comment (benoit-bst) [reply csv.cpp]: :speak_no_evil:

File: csv.cpp [hunk 2]
```diff
@@ -258,3 +258,17 @@ void remove_bom(std::fstream& stream){
...
+/**
+ * @brief check if row is empty
+ * Avoid to process empty row and row with only carriage return characters.
+ *
+ * @return true if row is empty
+ */
+bool CsvReader::row_is_empty(const std::vector<std::string>& row) {
+    if(row.empty() || (row.size() == 1 && row[0].empty()))
+        return true;
+    else
```
Comment (pbougue) [inline csv.cpp]: minor: most of the time we use brackets `} else {`

Comment (benoit-bst) [reply csv.cpp]: :white_check_mark:

**Pull request merged**